files/repo_names.txt

```diff
@@ -0,0 +1,32 @@
+sqre-codekit
```

Comment:
Do we maybe want to generate this list on the fly by asking GitHub instead?  Maybe not, since it probably isn't all repos.  Hm.